holidayextras/ui-toolkit
# Updating Payment Card to be more flexible with accepted props

#### What does this PR do? (please provide any background)
Allows you to pass extra props into the PaymentCard component.

This is required to replicate existing functionality which responds to onClick events.


Also process fixes:
- change to build the bundled version before release and s3 to make sure s3 gets the latest version.
- ignoring built docs bundle asset.

#### What tests does this PR have?
I've added tests for passing an `onClick` handler and and `id` 

#### How can this be tested?
Run the docs with `npm run docs`
In the browser edit the example for the PaymentCard
Check that your changes are reflected in the rendered DOM

#### Screenshots / Screencast
![](https://cldup.com/uH0C0qU2Y_-3000x3000.png)

#### What gif best describes how you feel about this work?
![](url)

---


#### Reviewers

**Review 1**

**Review 2** \*

**Review 3** _(optional)_

By adding a +1 you are confirming you have...
- Witnessed the work behaving as expected (this could be on the author's machine or screencast).
- Checked for coding anti-patterns.
- Checked for appropriate test coverage.
- Checked all the tests are passing.

\*  for HX this review must be completed by an SE, SA or Project Guru
